engine/updategraph/src/main/java/io/deephaven/engine/updategraph/UpdateGraphLock.java

```diff
@@ -14,8 +14,8 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.ArrayDeque;
 import java.util.Deque;
+import java.util.concurrent.ConcurrentLinkedDeque;
```

Comment:
Please get rid of the unused import introduced in the last PR:
`import io.deephaven.util.SafeCloseable;`